# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/qca2/hotfix-cmake4.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- qca-2.3.9/CMakeLists.txt.vanilla	2025-06-29 15:00:39.241679643 +0200
+++ qca-2.3.9/CMakeLists.txt	2025-06-29 15:00:50.424901439 +0200
@@ -15,10 +15,6 @@
 set(QCA_LIB_MINOR_VERSION "3")
 set(QCA_LIB_PATCH_VERSION "9")
 
-if(POLICY CMP0042)
-  cmake_policy(SET CMP0042 OLD)
-endif()
-
 option(BUILD_WITH_QT6 "Build with Qt 6" OFF)
 option(BUILD_TESTS "Create test" ON)
 option(BUILD_TOOLS "Compile mozcerts and qcatool" ON)
